### PR TITLE
Support Python3

### PR DESCRIPTION
**Pull request by alexpearce**
This PR adds Python 3 support to the Python bindings.

The changes are relatively minor, but I haven't been able to run the Python tests. Are there instructions for doing so? There's an `env.py` file that it looks like I should source somehow, and I think I should have a server running on `localhost`, but I don't know where to start. Any help would be appreciated!

### CODE REVIEW

File: bindings/python/libs/client/file.py [hunk 1]
```diff
@@ -42,7 +43,7 @@ def __exit__(self, type, value, traceback):
   def __iter__(self):
     return self
 
-  def next(self):
+  def __next__(self):
```
Comment (chrisburr) [inline bindings/python/libs/client/file.py]: In order to maintain compatibility with python 2 you need to add `next = __next__` to the class.

Comment (alexpearce) [inline bindings/python/libs/client/file.py]: Good spot, thanks! Fixed in 5164c6c.

**Comment by alexpearce** [comment]
Sorry to bother, but is there a chance this could be merged?

**Comment by simonmichal** [comment]
Hi Alex,

It seems to me that you have a typo in  bindings/python/libs/client/copyprocess.py : 40

you have:
> if 'success' in results:

should be:
> if 'status' in results:

Otherwise it looks good :-)
Anyway sorry for the delay!

Cheers,
Michal

**Comment by alexpearce** [comment]
Thanks for the review! I've fixed the typo.

**Pull request merged**